src/magic_folder/downloader.py

```diff
@@ -56,8 +56,17 @@
 from .snapshot import (
     create_snapshot_from_capability,
 )
-from .util.file import PathState, get_pathinfo, seconds_to_ns
-from .util.twisted import exclusively
+from .util.file import (
+    PathState,
+    get_pathinfo,
+    seconds_to_ns,
+)
+from .util.twisted import (
+    exclusively,
+)
```

Comment:
I'd slightly prefer to leave these as single lines. We've talked about (when we aren't under a deadline) using `black` and `isort` on the codebase. I've been trying (as much as practical), to at least format new code that way, to minimize the diff when we do. The original is how black would format these lines.

Reply:
okay, but all the rest is formatted like that. consistency is good.